.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: oD9uvSanHvlicy09GFJf8oUiJqec6ReHT
```

Comment:
It's not safe to have sensitive details like your coveralls repo token accessible on a public repo, what do you think ?